Space-Acres.app/Contents/Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>CFBundleExecutable</key>
+    <string>space-acres</string>
+    <key>CFBundleIdentifier</key>
+    <string>com.subspace.spaceacres</string>
+    <key>CFBundleName</key>
+    <string>Space Acres</string>
+    <key>CFBundleVersion</key>
+    <string>0.1.17</string>
```

Comment:
Is there a way to extract it automatically from `Cargo.toml`? We had a single place to update version until this point, having more places is not ideal.

Reply:
Not sure at this point. Will look into it.